packages/xo-web/src/xo-app/index.js

```diff
@@ -114,6 +116,29 @@ const BODY_STYLE = {
     signedUp: !!state.user,
   }
 })
+@provideState({
+  initialState: () => ({ checkXoaCount: 0 }),
+  effects: {
+    async forceRefreshXoaStatus() {
+      await clearXoaCheckCache()
+      await this.effects.refreshXoaStatus()
+    },
+    refreshXoaStatus() {
+      this.state.checkXoaCount += 1
+    },
+  },
```

Comment:
What about this, instead of the artificial dependency?
```js
effects: {
  initialize() {
    this.effects.refreshXoaStatus()
  }
  async refreshXoaStatus() {
    this.state.xoaStatus = await checkXoa()
  },
  async forceRefreshXoaStatus() {
    await clearXoaCheckCache()
    await this.effects.refreshXoaStatus()
  }
}
```

/cc @julien-f

Reply:
TBH I'm not found of either approaches, the artificial dependency is clumsy (maybe that could be fixed by providing Reaclette an explicit way to refresh a computed), but has the advantage of being lazy.

I cannot explain why but I'm not found of using `initialize`.

Reply:
Yes, `initialize` isn't lazy but here, we always want to check the status at least once, when the page loads anyway. So I think it's fine, and I find it more readable when you read the code for the first time.